components/mpas-albany-landice/src/mode_forward/mpas_li_time_integration.F

```diff
@@ -175,6 +178,8 @@ subroutine li_timestep(domain, err)
       end select
       err = ior(err,err_tmp)
 
+      ! accumulate fluxes for time averaging of fields sent to coupler
```

Comment:
For consistency with existing code, this should be put inside of a block loop (and the changes at lines 157-8 moved here).

Reply:
done